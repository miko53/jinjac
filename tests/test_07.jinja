Essai avec des constantes
une addition {{ 5 + 6 }} == 11
un autre calcul {{ 15- 4 }} == 11
un double {{ 0.156 *582.54 }} == 90.87624

