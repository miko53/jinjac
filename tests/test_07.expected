Essai avec des constantes
une addition 11 == 11
un autre calcul 11 == 11
un double 90.876240 == 90.87624

